app/src/androidTest/java/org/mozilla/fenix/ui/robots/SearchRobot.kt

```diff
@@ -82,8 +80,21 @@ class SearchRobot {
         onView(withText("Search engine settings")).perform(click())
     }
 
+    fun clickClearButton(){
+        clearButton().perform(click())
+    }
+
     class Transition {
         val mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
+
+        fun performSearch(searchTerm: String, interact: TabsRobot.() -> Unit): TabsRobot.Transition{
+            mDevice.waitForIdle()
```

Comment:
unfortunately, `mDevice.waitForIdle()` by itself doesn't seem to do much.  We still have several statements like this in our tests that need swapping out, but waiting on the existence of a specific text string is usually more effective, i.e.:  https://github.com/mozilla-mobile/fenix/blob/master/app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt#L37-L40

Reply:
performSearch is also not neccessary anymore, there is a method in Browser Robot that does the same thing. I'll remove this too.